test/unit/ResourceTypeRegistry.js

```diff
@@ -36,6 +54,24 @@ describe("ResourceTypeRegistry", function() {
     it("should be a getter/setter for a type",
       makeGetterSetterTest(description, "mytypes", "type", true)
     );
+
+    it("should merge descriptionDefaults into resource description", () => {
+      registry = new ResourceTypeRegistry([], {
+        info: "provided as default"
+      });
+
+      registry.type("someType", {});
+      expect(registry.type("someType").info).to.equal("provided as default");
```

Comment:
We could add another assertion here to check that the global defaults are merged in too (and maybe rename the test accordingly).